tools/cldr-code/src/test/resources/org/unicode/cldr/tool/KeyboardFlatten/broken-import-missing.xml

```diff
@@ -45,31 +45,31 @@
     </keys>
 
     <layers formId="iso">
-        <layer modifier="none">
+        <layer modifiers="none">
             <row keys="c-tikka 1 2 3 4 5 6 7 8 9 0 minus equals" />
             <row keys="q w e r t y u i o p g-tikka h-maqtua" />
             <row keys="a s d f g h j k l semi-colon hash" />
             <row keys="z-tikka z x c v b n m comma period slash" />
             <row keys="space" />
         </layer>
 
-        <layer modifier="shift">
+        <layer modifiers="shift">
             <row keys="C-tikka bang double-quote euro dollar percent caret amp open-paren close-paren underscore plus" />
             <row keys="Q W E R T Y U I O P G-tikka H-maqtua" />
             <row keys="A S D F G H J K L colon at tilde" />
             <row keys="Z-tikka Z X C V B N M open-angle close-angle question" />
             <row keys="space" />
         </layer>
 
-        <layer modifier="altR">
+        <layer modifiers="altR">
             <row keys="grave gap gap pound gap gap gap gap gap gap gap gap gap" />
             <row keys="gap gap e-grave gap gap gap u-grave i-grave o-grave gap open-square close-square" />
             <row keys="a-grave gap gap gap gap gap gap gap gap gap gap gap" />
             <row keys="backslash gap gap gap gap gap gap gap gap gap gap" />
             <row keys="space" />
         </layer>
 
-        <layer modifier="altR-shift">
+        <layer modifiers="altR-shift">
```

Comment:
Hyphen? I don't think that is the syntax described above, and the DTD pattern does not allow hyphens (though maybe it should, together with underscores).